charts/operator/templates/configmap.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if .Values.airgapped }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "name" . }}-chart-configmap
+data:
+  {{- range .Values.charts }}
+    {{ .name }}.b64: {{ $.Files.Get .path | b64enc }}
```

Comment:
we are explicitly annotating with `.b64` - in operator code, we check if the file has this extension and handle accordingly